nv_dds.h

```diff
@@ -195,6 +195,12 @@ class CDDSImage {
         return m_images[face];
     }
 
+    const CTexture &get_surface() const {

```

Comment:
```suggestion
    const CTexture &get_surface(unsigned int layer) const {
```

Reply:
while similar to `get_cubemap_face` it would state that you expect an array texture. (with a 2D texture being a special case

Reply:
Rebased !
Not really matters to me, I'm not keen on either solution.